editoast/src/core/stdcm.rs

```diff
@@ -90,6 +93,15 @@ pub struct LightWorkSchedule {
     pub track_ranges: Vec<UndirectedTrackRange>,
 }
 
+/// Lighter description of a work schedule with only the informations relevant for core
+#[derive(Debug, Serialize)]
+pub struct LightTemporarySpeedLimit {
+    // Speed limitation in km/h
+    pub speed_limit: i64,
```

Comment:
- Use `///` for documentation.
- Speed unit is `m/s` using `f64`

```suggestion
    /// Speed limitation in m/s
    pub speed_limit: f64,
```